source/Symbol/SwiftASTContext.cpp

```diff
@@ -8543,6 +8627,8 @@ SwiftASTContext::GetIndexOfChildMemberWithName (void* type, const char *name,
 uint32_t
 SwiftASTContext::GetIndexOfChildWithName (void* type, const char *name, bool omit_empty_base_classes)
 {
+    VALID_OR_RETURN(0);
```

Comment:
It'd be nice if there were an invalid value we could use here, even just `UINT32_MAX`.